CHANGELOG.md

```diff
@@ -67,16 +67,16 @@ Adding a new version? You'll need three changes:
 
 ### Added
 
-  - Added `HTTPRoute` support for `CombinedRoutes` feature. When enabled,
+- Added `HTTPRoute` support for `CombinedRoutes` feature. When enabled,
```

Comment:
Not a comment, but wanted to know what is the tool to format the markdown docs

Reply:
I'm using https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint which is pretty helpful but you might need to disables some of the checks as this repo was not linted all that much before :)